### DISCUSSION TITLE
Editing images on the site

### DISCUSSION THREAD

**Discussion by aukc1970** [givanz/Vvveb#237] [General]
Good afternoon! Editing images on the site is very important, I have a suggestion to add a Cropper image editing script to your CMS.
Here is the link
https://fengyuanchen.github.io/cropperjs/

**Comment by givanz** [comment]
Hi

I added media image editing in `latest.zip` update https://demo.vvveb.com/admin/index.php?module=media/media 

Library used https://github.com/scaleflex/filerobot-image-editor

**Reply by aukc1970** [reply]
Thank you very much!
